fletcher/algorithms/string_builder_nojit.py

```diff
@@ -0,0 +1,222 @@
+import math
```

Comment:
Do we want to keep this as the jitted-version now also runs without a JIT?

Reply:
@windiana42 🔝

Reply:
The string_builder_nojit was very helpful during development since even in NUMBA_DISABLE_JIT=1 mode, you easily kill your debugging session with segmentation faults.

Reply:
Ok, then I'll merge and let's see whether we use this occasionally. If this bitrots, we can remove it later. This way we have it at least in the master history.